server/database/models/UserRepository.js

```diff
@@ -0,0 +1,113 @@
+const AbstractRepository = require("./AbstractRepository");
+
+class UserRepository extends AbstractRepository {
+  constructor() {
+    // Call the constructor of the parent class (AbstractRepository)
+    // and pass the table name "user" as configuration
+    super({ table: "user" });
+    this.avatar_image = "client/src/assets/images/avatar/Avatar-basic.svg";
+  }
+
+  // The C of CRUD - Create operation
+
+  async create(user) {
+    const avatarImage = this.avatar_image;
+    // Execute the SQL INSERT query to add a new user to the "user" table
+    const [result] = await this.database.query(
+      `INSERT INTO ${this.table} (
+        created_at,
```

Comment:
dead code because your database already know a specific method to affect a correct date